packages/terra-navigation-side-menu/src/_MenuItem.jsx

```diff
@@ -1,18 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
+import { injectIntl, intlShape } from 'react-intl';
 import KeyCode from 'keycode-js';
 import ChevronRight from 'terra-icon/lib/icon/IconChevronRight';
 
 import styles from './MenuItem.module.scss';
 
+const isSafari = (navigator.userAgent.indexOf('Safari') !== -1 && navigator.userAgent.indexOf('Chrome') === -1);
```

Comment:
Let's add a comment that explains that we only want to do this type of thing sparingly and this is specifically for an accessibility issue between browsers.  This should not be the pattern that teams should use, etc.

Reply:
added https://github.com/cerner/terra-framework/pull/761/commits/3fc811e96c6bcc064e1bc211bf767438514c6cd9, take a look